.github/workflows/ci.yml

```diff
@@ -116,7 +120,8 @@ jobs:
 
       # Tests
       - name: Pytest
-        run: pytest -n auto --dist loadscope --cov --cov-report xml
+        # FIXME: Add `-n auto` again to allow for parallelism here.
+        run: pytest --cov --cov-report xml
```

Comment:
From experience, the `pytest-cov` runs after module initialization. So parts of any imported module that run directly after import will happen before the coverage and thus won't be a part of the coverage. I used `coverage run pyest ...`.